core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSpec.scala

```diff
@@ -290,7 +290,7 @@ class SubmissionSpec(_system: ActorSystem) extends TestKit(_system)
     withDataOp { dataSource =>
       val execServiceCluster: ExecutionServiceCluster = MockShardedExecutionServiceCluster.fromDAO(executionServiceDAO, dataSource)
 
-      val config = SubmissionMonitorConfig(250.milliseconds, trackDetailedSubmissionMetrics = true)
+      val config = SubmissionMonitorConfig(250.milliseconds, trackDetailedSubmissionMetrics = true, 20000)
```

Comment:
nit: anywhere you've hardcoded 20000, should it pull from src/test/resources/reference.conf instead? This isn't a big deal, don't hold the PR for this, it might avoid confusion in the future

Reply:
[This](https://github.com/broadinstitute/rawls/blob/3f3fff74701a4618493bb4ad04d8f5a6bcbfc79e/core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/BootMonitors.scala#L64-L68) was the only place I found where we actually instantiate this class with config values. The tests all seem to be hard-coded, strange as that may seem.

Reply:
huh. Well then, consistency is good - no objections!